packages/account/src/Components/trading-assessment/trading-assessment-radio-buttons.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Field } from 'formik';
+import { Text, RadioGroup } from '@deriv/components';
+
+const TradingAssessmentRadioOption = ({ text, list, onChange, values, form_control, setEnableNextSection }) => {
+    React.useEffect(() => {
+        if (values[form_control]) {
+            setEnableNextSection(true);
```

Comment:
Can we just do `setEnableNextSection(!!values[form_control])` instead of this condition?